daffodil-cli/src/it/scala/org/apache/daffodil/unparsing/TestCLIUnparsing.scala

```diff
@@ -259,13 +259,13 @@ class TestCLIunparsing {
 
     try {
 
-      shell.sendLine(String.format("""echo "<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>" | %s -v unparse -s %s --root e1""", Util.binPath, testSchemaFile))
+      shell.sendLine(String.format(Util.echoN("\"<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>\"") + "| %s -v unparse -s %s --root e1", Util.binPath, testSchemaFile))
       shell.expectIn(1, contains("[info]"))
 
-      shell.sendLine(String.format("""echo "<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>" | %s -vv unparse -s %s --root e1""", Util.binPath, testSchemaFile))
+      shell.sendLine(String.format(Util.echoN("\"<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>\"") + "| %s -vv unparse -s %s --root e1", Util.binPath, testSchemaFile))
       shell.expectIn(1, contains("[compile]"))
 
-      shell.sendLine(String.format("""echo "<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>" | %s -vvv unparse -s %s --root e1""", Util.binPath, testSchemaFile))
+      shell.sendLine(String.format(Util.echoN("\"<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>\"") + "| %s -vvv unparse -s %s --root e1", Util.binPath, testSchemaFile))
```

Comment:
Looks like you've only updated this single test `test_3580_CLI_Unparsing_SimpleUnparse_verboseMode`. This test is passing in the github actions on windows, so it does seem like echoN is the right fix.

But there are other tests that are failing that need the same change:
* test_3528_CLI_Unparsing_SimpleUnparse_stdin2
* test_3529_CLI_Unparsing_SimpleUnparse_stdin3
* test_3578_CLI_Unparsing_SimpleUnparse_defaultRoot
* test_3583_CLI_Unparsing_SimpleUnparse_rootPath

Also, these two tests use echo but happen to work because they are negative tests that fail for reasons other than echo issues. But for correctness these should use echoN too:
* test_3662_CLI_Unparsing_badSchemaPath
* test_3579_CLI_Unparsing_negativeTest